### ISSUE TITLE
pypi is outdated

### ISSUE THREAD

**Issue by foarsitter** [cookiecutter/cookiecutter-django#5350]
## What happened?

Came across the `setup.py` and wondered why it exists since we do not deploy to pypi, do we? Turns out we did in the past but the last update is of januari 2018. 

## What should've happened instead?

I would propose to remove the package for pypi completely since it contains outdated information and starting releasing again wouldn't be of much benefit. 

## Additional details
Since setup.py has no purpose I'm planning to merge it with pyproject.toml.

**Comment by luzfcb** [comment]
The purpose of the pypi package is only to ensure that we own the name and prevent it from being used by a malicious actor.
That said, it might be a good idea to update it and make its purpose clearer.

**Comment by foarsitter** [comment]
We can create a entrypoint `cookiecutter-django` which invokes cookiecutter `cookiecutter https://github.com/cookiecutter/cookiecutter-django --version $version`?

**Comment by SaboorNisha** [comment]
You're right—if we no longer deploy to PyPI and haven't updated the package since January 2018, it makes sense to consider removing it. Merging setup.py with pyproject.toml sounds like a good approach to streamline the project configuration. 
This change could also help reduce confusion for contributors who might assume the PyPI deployment is still active.

**Comment by dominiqueplante** [comment]
It looks like this should be closed - it looks like it was done as part of Release 2024.09.06

cc @SaboorNisha & @foarsitter & @luzfcb

**Comment by browniebroke** [comment]
> It looks like this should be closed - it looks like it was done as part of Release 2024.09.06

Where do you see that? I think the latest release is from 2018: https://pypi.org/project/cookiecutter-django/#history

**Comment by dominiqueplante** [comment]
The specific commit where setup.py was removed is here -> https://github.com/cookiecutter/cookiecutter-django/commit/d6f18f04a9a2f3c0b7c85b783b8f360fbcb453e5#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7
